### PR TITLE
chore: update README.md

### PR DESCRIPTION
**Pull request by HarshMN2345**
made it easy to understand and improved grammar

## Checklist

- [ ] `.changeset`
- [ ] unit tests
- [ ] integration tests
- [x] docs

OR

- [ ] This PR is not applicable for the checklist

### CODE REVIEW

**Comment by charIeszhao** [comment]
@gao-sun Triaged

File: README.md [hunk 1]
```diff
@@ -14,27 +14,28 @@
 
 Logto[^info] is a cost-effective open-source alternative to Auth0. It offers a seamless developer experience and is well-suited for individuals and growing companies.
 
-🧑‍💻 **A frontend-to-backend identity solution**
+🧑‍💻 **Comprehensive Frontend-to-Backend Identity Solution**
```
Comment (gao-sun) [inline README.md:17]:
```suggestion
🧑‍💻 **Comprehensive frontend-to-backend identity solution**
```

File: README.md [hunk 2]
```diff
@@ -14,27 +14,28 @@
...
 
-- OIDC-based authentication and RBAC authorization.
-- Passwordless sign in and much more diverse options, including Email, Phone number, Username, Google, Facebook and other social sign in methods.
-- Beautiful UI components with customizable CSS to fit your business needs.
+- Enables OIDC-based authentication and RBAC authorization.
+- Supports passwordless sign-in, along with various options like Email, Phone number, Username, Google, Facebook, and other social sign-in methods.
+- Offers beautiful UI components with customizable CSS to suit your business needs.
 
-📦 **Out-of-the-box infrastructure**
+📦 **Out-of-the-Box Infrastructure**
```
Comment (gao-sun) [inline README.md:23]:
```suggestion
📦 **Out-of-the-box infrastructure**
```

File: README.md [hunk 3]
```diff
@@ -14,27 +14,28 @@
...
 
-- A ready-to-use management API can serve as your authentication provider, eliminating the need for extra implementation.
-- SDKs that can integrate your apps with Logto quickly, multi-platform and language compatible, tailored to your development environment.
-- Flexible connectors, scalable with community contributions, customizable with SAML, OAuth, and OIDC protocols.
+- Includes a ready-to-use management API, serving as your authentication provider, thus eliminating the need for extra implementation.
+- Provides SDKs that seamlessly integrate your apps with Logto across multiple platforms and languages, tailored to your development environment.
+- Offers flexible connectors that can be scaled with community contributions and customized with SAML, OAuth, and OIDC protocols.
 
-💻 **Enterprise-ready solutions**
+💻 **Enterprise-Ready Solutions**
```
Comment (gao-sun) [inline README.md:29]:
```suggestion
💻 **Enterprise-ready solutions**
```

File: README.md [hunk 4]
```diff
@@ -14,27 +14,28 @@
...
 
-- RBAC to control your resource through scalable role authorization for diverse use cases.
-- User management and audit Logs to understand identity related user info and keep your security on track.
-- We are currently working on SSO, Organizations and MFA. Stay tuned.
+- Implements RBAC for scalable role authorization, catering to a wide range of use cases.
```
Comment (gao-sun) [inline README.md:31]:
```suggestion
- Implements Role-Based Access Control (RBAC) for scalable role authorization, catering to a wide range of use cases.
```

**Review by gao-sun — APPROVED** [review]
thank you. i'll tweak some details before merging since we have the convention of commit messages and using sentence case

**Pull request merged**